app/src/main/java/ingsw/group1/findmyphone/contacts/SMSContactSwipeCallback.java

```diff
@@ -0,0 +1,126 @@
+package ingsw.group1.findmyphone.contacts;
+
+import android.content.Context;
+import android.content.Intent;
+import android.graphics.Canvas;
+import android.view.View;
+
+import androidx.annotation.NonNull;
+import androidx.core.content.ContextCompat;
+import androidx.recyclerview.widget.ItemTouchHelper;
+import androidx.recyclerview.widget.RecyclerView;
+import androidx.room.Ignore;
+
+import com.google.android.material.snackbar.Snackbar;
+
+import ingsw.group1.findmyphone.R;
+import ingsw.group1.findmyphone.activity.ContactListActivity;
+import ingsw.group1.findmyphone.activity.ModifyContactActivity;
+import it.xabaras.android.recyclerview.swipedecorator.RecyclerViewSwipeDecorator;
+
+/**
+ * Class using in the {@link ContactListActivity} to do an action of contact deletion
+ * after a swipe on an item in the contacts list.
+ * When user swipes an item to LEFT or RIGHT, it is invoked {@link SMSContactSwipeCallback#onSwiped(RecyclerView.ViewHolder, int)}
+ * to delete (or undo the deletion of a contact) or to modify that item
+ * and {@link SMSContactSwipeCallback#onChildDraw(Canvas, RecyclerView, RecyclerView.ViewHolder, float, float, int, boolean)}
+ * to show a colorful background and an icon.
+ *
+ * @author Giorgia Bortoletti
+ */
+public class SMSContactSwipeCallback extends ItemTouchHelper.SimpleCallback {
+
+    private SMSContactRecyclerAdapter contactAdapter;
+
+    /**
+     * Constructor
+     *
+     * @param adapter
+     */
+    public SMSContactSwipeCallback(SMSContactRecyclerAdapter adapter) {
+        super(0, ItemTouchHelper.LEFT | ItemTouchHelper.RIGHT);
+        contactAdapter = adapter;
+    }
+
+    /**
+     * It is necessary to override but it is never used.
+     */
+    @Ignore
+    @Override
+    public boolean onMove(@NonNull RecyclerView recyclerView, @NonNull RecyclerView.ViewHolder viewHolder, @NonNull RecyclerView.ViewHolder target) {
+        return false;
```

Comment:
This is the callback for when the user moves an item in the list (?) Maybe we could block this action from the view's properties

Reply:
Yes, I'll add documentation. 
In this moment this action is blocked and user can't move an item.

Reply:
Good👌